docs/images.md

```diff
@@ -0,0 +1,24 @@
+# Images and their usage in MLRun
+
+Every release of MLRun includes several images for different usages. The build and the infrastructure images are described, and located, in the [README](https://github.com/mlrun/mlrun/blob/development/dockerfiles/README.md). They are also published to [dockerhub](https://hub.docker.com/u/mlrun) and [quay.io](https://quay.io/organization/mlrun).
+
+## Using images
+
+See [Kubernetes Jobs & Images](./runtimes/mlrun_jobs.ipynb)
+
+## MLRun images and how to build them 
+
+See [README](https://github.com/mlrun/mlrun/blob/development/dockerfiles/README.md).
+
+## MLRun images and external docker images
+
+There is no difference in the usage between the MLRun images and external docker images. However, MLRun images:
+- Resolve auto tags: If you specify ```image="mlrun/mlrun"``` the API fills in the tag by the client version, e.g. changes it to `mlrun/mlrun:0.9.2`. So, if the client gets upgraded you'll automatically get a new API image. 
+- Append the registry prefix, loading the image from the datanode registry, except for any third party k8s installations. This pulls the image more quickly, and also supports air-gapped sites. When you specify an MLRun image, for example `mlrun/mlrun:0.9.2`, the actual image used is similar to `datanode-registry.iguazio-platform.app.vm`.
```

Comment:
1. You need to mention this is only happening on the enterprise version, for installations where the datanode registry exists (e.g. on third party k8s it does not)
2. the example you give should be changed to `datanode-registry.iguazio-platform.app.vm/mlrun/mlrun:0.9.2`

Reply:
fixed